interop-tests/src/main/protobuf/crdt-example.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
```

Comment:
Why is this file in this PR?

Reply:
Without the change in this PR, this .proto would generate gRPC javadsl service code that referred to a `Crdt-example` outer class, which wouldn't compile, reproducing the original problem.